src/Mailer/Handler/SendOwnershipMailCommandHandler.php

```diff
@@ -9,65 +9,67 @@
 use CultuurNet\UDB3\Mailer\Handler\Helper\OwnershipMailParamExtractor;
 use CultuurNet\UDB3\Mailer\Mailer;
 use CultuurNet\UDB3\Mailer\MailsSentRepository;
+use CultuurNet\UDB3\Mailer\Ownership\RecipientStrategy\RecipientStrategy;
 use CultuurNet\UDB3\Model\ValueObject\Identity\Uuid;
 use CultuurNet\UDB3\Model\ValueObject\Web\EmailAddress;
+use CultuurNet\UDB3\Ownership\Repositories\OwnershipItem;
 use CultuurNet\UDB3\Ownership\Repositories\OwnershipItemNotFound;
 use CultuurNet\UDB3\Ownership\Repositories\Search\OwnershipSearchRepository;
 use CultuurNet\UDB3\ReadModel\DocumentDoesNotExist;
-use CultuurNet\UDB3\User\UserIdentityResolver;
+use CultuurNet\UDB3\User\UserIdentityDetails;
 use DateTimeImmutable;
 use Psr\Log\LoggerInterface;
 use Twig\Environment as TwigEnvironment;
 use Twig\Error\LoaderError;
 use Twig\Error\RuntimeError;
 use Twig\Error\SyntaxError;
 
-class SendOwnershipMailCommandHandler implements CommandHandler
+final class SendOwnershipMailCommandHandler implements CommandHandler
 {
     private const SUBJECT_OWNERSHIP_REQUESTED = 'Beheers aanvraag voor organisatie {{ organisationName }}';
     private const TEMPLATE_OWNERSHIP_REQUESTED = 'ownershipRequested';
 
     private Mailer $mailer;
     private MailsSentRepository $mailsSentRepository;
-    private UserIdentityResolver $identityResolver;
     private TwigEnvironment $twig;
     private OwnershipSearchRepository $ownershipSearchRepository;
     private OwnershipMailParamExtractor $paramExtractor;
+    private RecipientStrategy $sendToCreatorOfOrganisation;
     private LoggerInterface $logger;
 
     public function __construct(
         Mailer $mailer,
         MailsSentRepository $mailsSentRepository,
-        UserIdentityResolver $identityResolver,
         TwigEnvironment $twig,
         OwnershipSearchRepository $ownershipSearchRepository,
         OwnershipMailParamExtractor $paramExtractor,
+        RecipientStrategy $sendToCreatorOfOrganisation,
```

Comment:
I guess this parameter name is no longer correct because of using the combined strategy?